streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBWindowStoreTest.java

```diff
@@ -107,12 +107,7 @@
 
     private WindowStore<Integer, String> createWindowStore(final ProcessorContext context, final boolean retainDuplicates) {
         final WindowStore<Integer, String> store = Stores.windowStoreBuilder(
-            Stores.persistentWindowStore(
-                windowName,
-                retentionPeriod,
-                numSegments,
-                windowSize,
-                retainDuplicates),
+            Stores.persistentWindowStore(windowName, retentionPeriod, windowSize, retainDuplicates, segmentInterval),
```

Comment:
Oops! I found a couple of spots where we were using `numSegments` when we meant `segmentInterval`. Clearly, the segment size doesn't matter for these tests (because it was wrong), so maybe we should use the non-parameterized version....

Reply:
Makes sense.

Reply:
Actually, I re-read this section. It was indeed incorrect before, but it still worked because the segmentInterval we specified (3ms) divides the one we tested for (60_000ms). Nevertheless, the tests do depend on a specified segmentInterval, so I think we should specify it here.